packages/flutter/lib/src/widgets/interactive_viewer.dart

```diff
@@ -412,79 +428,10 @@ class InteractiveViewer extends StatefulWidget {
   ///  * [TextEditingController] for an example of another similar pattern.
   final TransformationController? transformationController;
 
-  /// Returns the closest point to the given point on the given line segment.
-  @visibleForTesting
-  static Vector3 getNearestPointOnLine(Vector3 point, Vector3 l1, Vector3 l2) {
```

Comment:
These methods that I removed were involved in my previous iteration of correcting an invalid matrix, which in this PR I have generalized to work with a rotated matrix as well.